feat: add has_root_access to netapp vol export rules (#11105)
terraform-providers/terraform-provider-azurerm: website/docs/r/netapp_volume.html.markdown

* `unix_read_write` - (Optional) Is the file system on unix read and write?

> * `allow_root_access` - (Optional) Has root access to the volume?

Reviewer: presumably this should be:

```suggestion
* `allow_root_access` - (Optional) Is root access permitted to this volume?
```
